src/account.js

```diff
@@ -86,6 +87,23 @@ const Account = stampit().compose(Request)
     },
 
     /**
+    * A convenience method for authenticating with a social media token.
+
+    * @memberOf Account
+    * @instance
+
+    * @param {Object} payload
+    * @param {String} payload.backend
+    * @param {String} payload.access_token
+    * @returns {Promise}
+
+    */
+    socialLogin(payload = {}) {
```

Comment:
maybe we should do something similar like in case of `login` and `setAccountKey` ?